roofit/roofitcore/src/RooMultiPdf.cxx

```diff
@@ -86,4 +86,5 @@ void RooMultiPdf::getParametersHook(const RooArgSet *nset, RooArgSet *list, bool
 
    list->removeAll();
    getCurrentPdf()->getParameters(nset, *list, stripDisconnected);
+   list->add(*x);
```

Comment:
Thanks for the PR and addressing the comments I have done offline! Could you also make the same change to the `RooMultiReal` to keep things consistent? Thanks!